projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -257,6 +257,9 @@ control_service_deploy_testing_data_pipelines:
     - pip install --upgrade pip && pip install awscli
     - export FRONTEND_TAG=latest
     - export DESIRED_VERSION=v3.12.0 # helm version
+    - curl -LO https://dl.k8s.io/release/v1.27.2/bin/linux/amd64/kubectl
+    - chmod +x ./kubectl
+    - mv ./kubectl /usr/bin/kubectl
     - curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
     - bash -ex ./projects/control-service/cicd/deploy-testing-pipelines-service.sh
```

Comment:
How come we find about this now, can you add it to the PR description? For example, link the cicd job indicating the missing command? 
The cause is with the image change from proum/aws-iam-authenticator in Jan, or something else?

Reply:
there should be a cicd job failure as a result from such a regression. did we find such?

Reply:
Probably no CICD failure: `kubectl` is used only when `RUN_ENVIRONMENT_SETUP` is set to 'y' (yes) in deploy-testing-pipelines-service.sh

And it's never really been set to yes in the Gitlab. I ran it once when initially setting up the environment in the beginning and I ran it likely from my machine.